content/jobs/tanks/dark-knight/advanced-guide.md

```diff
@@ -114,34 +123,57 @@ Living Shadow's base strength is calculated differently to the player. In partic
 
 - Living Shadow's strength includes all strength bonuses from gear, as normal
 
-To be more specific, Living Shadow's base strength (before strength buffs like potions) is calculated by the following formula (where `baseStrength` is `440` at level 100):
+To be more specific, Living Shadow's base strength (before strength buffs like potions) is calculated by the following formula (where \(\rm{baseStrength}\) is \(440\) at level 100):
 
-`baseStrength + strengthProvidedByGear + 2`
+\[
+\begin{aligned}
+\rm{baseStrength} + \rm{strengthProvidedByGear} + 2
+\end{aligned}
+\]
 
-Living Shadow's weapon damage multplier is calculated differently to the player. In particular:
+Living Shadow's weapon damage multiplier is calculated differently to the player. In particular:
 
-- While players would use a weapon damage multplier of `115`, Living Shadow uses a weapon damage multiplier of `100`
+- While players would use a weapon damage multiplier of \(115\), Living Shadow uses a slightly lower weapon damage multiplier of \(100\)
 
 - This is typical for all 'pet actions', and generally true for any damage that does not come 'from' the player
 
 This changes the weapon damage multiplier formula from the following:
 
-`floor(baseStrength * 100 / 1000 + weaponDamage) / 100`
+\[
+\begin{aligned}
+\frac{\left \lfloor{\frac{\rm{baseStrength} \times 115}{1000} + \rm{weaponDamage}}\right \rfloor}{100}
+\end{aligned}
+\]
+
+To the following (where \(\rm{baseStrength}\) is \(440\) at level 100):
 
-To the following (where `baseStrength` is 440 at level 100):
```

Comment:
This was incorrect, I accidentally had the formulae the wrong way around. The player uses `115` as the job mod, pet actions use `100`.